### DISCUSSION TITLE
[v5] Use Adonis with Vite

### DISCUSSION THREAD

**Discussion by thefrana** [adonisjs/core#2517] [Help]
Hi @RomainLanz,
I just encountered your issue https://github.com/vitejs/vite/issues/1662. Did you manage to make it work? I am finding a way to seamlessly connect Svelte (or basically any frontend lib) and Adonis v5. I thought it would simplify a lot things if I use Vite for both of them. My idea is to route all URLs to some Svelte router (and preferably incorporate SSR) except of those which starts with `/api`. Those would be handled by Adonis controllers.

Do you have any idea that could help me?

Thank you very much

**Comment by mauriciocc** [comment]
Hey @lukeeno 

Made a very simple sample app wiring Adonis v5 + Vite: 

https://github.com/boolb/adonis-vite-demo

Hope it helps 😃
